### ISSUE TITLE
Lissi: Research on how it works, API

### ISSUE THREAD

**Issue by veitmo** [amosproj/amos2022ss04-digital-identity#28]
## User story
1. As a {}
2. I want / need {}
3. So that {}

## Acceptance criteria
* Amongst others, have had a look at the links provided by Adorsys (see Teams Channel "Links")
* have a look at most important upcoming functionalities:
* send personal information to API
* how is the QR-code created and delivered?
* how can an update of personal information be triggered and what are the steps to perform an update?

## Definition of done (DoD)
* Added only after week 5
* The same for all features
* Here goes the project specific part

## DoD general criteria
* Feature has been fully implemented
* Feature has been merged into the mainline
* All acceptance criteria were met
* Product owner approved features
* All tests are passing
* Developers agreed to release

**Comment by jackDS008** [comment]
**Help needed:** I got everything besides how the QR code is generated.

I only got a QR code for [verifying the e-mail-adress](https://lissi.id/authentification). Is it the right one?

Unfortunately, I didn't find any information on how to generate a QR code, which we can request with our backend.

**Comment by veitmo** [comment]
@jackDS008 
if I understood Mario correctly, the Lissi API provides a URL after the creation of the Digital Identity. This URL has to be converted to a QR-code (we can choose a tool to do that) and the QR code is then sent to the email-adress of the new DI

**Comment by veitmo** [comment]
@jackDS008 
this should also be mentioned in the notes of the meeting with Mario. If you have any further questions, reach out to us :)

**Comment by jackDS008** [comment]
Thanks for helping me. I found a hint in the notes of the meeting and finally found the right endpoint. It's the endpoint "create-invitation".
